x/concentrated-liquidity/simulation/sim_msgs.go

```diff
@@ -1,17 +1,68 @@
 package simulation
 
-// func RandomMsgCreateConcentratedPool(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*clmodeltypes.MsgCreateConcentratedPool, error) {
-// 	return nil, nil
-// }
+import (
+	"errors"
+	"fmt"
+	"math/rand"
 
-// func RandMsgCreatePosition(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*cltypes.MsgCreatePosition, error) {
-// 	return nil, nil
-// }
+	sdk "github.com/cosmos/cosmos-sdk/types"
 
-// func RandMsgWithdrawPosition(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*cltypes.MsgWithdrawPosition, error) {
-// 	return nil, nil
-// }
+	legacysimulationtype "github.com/cosmos/cosmos-sdk/types/simulation"
 
-// func RandMsgCollectFees(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*cltypes.MsgCollectFees, error) {
-// 	return nil, nil
-// }
+	osmosimtypes "github.com/osmosis-labs/osmosis/v14/simulation/simtypes"
+	clkeeper "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity"
+	clmodeltypes "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/model"
+	cltypes "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/types"
+)
+
+var PoolCreationFee = sdk.NewInt64Coin("stake", 10_000_000)
+
+func RandomMsgCreateConcentratedPool(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*clmodeltypes.MsgCreateConcentratedPool, error) {
+	minExponentAtOneValue := cltypes.ExponentAtPriceOneMin.Int64()
+	maxExponentAtOneValue := cltypes.ExponentAtPriceOneMax.Int64()
+
+	// generate random values from -13 to 1 (current accepted range: -12 to -1)
+	exponentAtPriceOne := sdk.NewInt(minExponentAtOneValue + rand.Int63n(maxExponentAtOneValue-minExponentAtOneValue+1))
```

Comment:
Im not quite understanding this. Why don't we just pick a random int between
`minExponentAtOneValue - 1` and `maxExponentAtOneValue + 2`?

Reply:
oh yea forgot about that. It was randomizing(-12, -1). Fixed this to rand(-13 ,1)